docs/devel/flaky-tests.md

```diff
@@ -11,32 +11,29 @@ There is a testing image ```brendanburns/flake``` up on the docker hub.  We will
 
 Create a replication controller with the following config:
 ```yaml
-id: flakecontroller
+apiVersion: v1beta3
 kind: ReplicationController
-apiVersion: v1beta1
-desiredState:
+metadata:
+  labels:
+    name: flake
+  name: flakecontroller
+spec:
   replicas: 24
-  replicaSelector:
+  selector:
     name: flake
-  podTemplate:
-    desiredState:
-      manifest:
-        version: v1beta1
-        id: ""
-        volumes: []
-        containers:
-        - name: flake
-          image: brendanburns/flake
-          env:
-          - name: TEST_PACKAGE
-            value: pkg/tools
-          - name: REPO_SPEC
-            value: https://github.com/GoogleCloudPlatform/kubernetes
-      restartpolicy: {}
-    labels:
-      name: flake
-labels:
-  name: flake
+  template:
+    metadata:
+      labels:
+        name: flake
```

Comment:
too many "name: flake" here :)
@bgrant0607 mentioned that we now populate the RC selector and labels from the pod template's labels by default.
We should take advantage of that and can skip the RC selector and labels.